ml-agents/mlagents/trainers/trainer_util.py

```diff
@@ -108,3 +109,31 @@ def initialize_trainers(
                 "brain {}".format(brain_name)
             )
     return trainers
+
+
+def load_config(trainer_config_path: str) -> Dict[str, Any]:
+    try:
+        with open(trainer_config_path) as data_file:
+            return _load_config(data_file)
+    except IOError:
+        raise UnityEnvironmentException(
+            f"Config file could not be found at {trainer_config_path}."
+        )
+    except UnicodeDecodeError:
+        raise UnityEnvironmentException(
+            f"There was an error decoding Config file from {trainer_config_path}. "
```

Comment:
I made these exceptions a bit more generic since they're used for both Trainer and Sampler config loading.

Reply:
In that case I'd recommend renaming the argument to just `config_path`